### PR TITLE
Milestone 3: TodoList

### PR DESCRIPTION
**Pull request by enaburekhan**
**TodoList built with vanilla javascript** 
**In this project:**

- Set up the project using webpack as the javascript bundler
- Create the Todo-List layout using HTML/CSS
- Create the dom module
- Create the project and task class and append the required attributes
- Separate each module/class by placing them in their respective files
- Create export and import on the required files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Todo-List",
+  "version": "1.0.0",
+  "description": "A Todo list project using javascript",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack serve --open"
```
Comment (addod19) [inline package.json]: [Optional]
- Kindly replace => "start": "webpack serve --open" with => "start": "webpack-dev-server --mode development --open"

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @enaburekhan and @Haywhizzz, great job on this project :+1: PR has a meaningful title and a short description :100: The README file is descriptive :fireworks: The UI is awesome :smile: However, there are a few issues that need to be addressed, kindly check the comments below:

- Kindly replace all alert() statements with a custom modal to inform users of any feedback from the application. It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead
- Our Task Object is missing the "title" and the "description". Currently, we have a name and what we actually need is a title and a description
- Kindly remove .vscode from the project, files not related to the project should not be committed and added to .gitignore (e.g. .vscode)

[Optional]
- It would be great to have an online version of the app so that when the app fails to run locally we can test it using the online version

Once these are fixed, kindly submit another code review request :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 
- Good work applying the changes requested by the previous TSE.

## However, there are some changes required 👀 🧰 🔨 🔧 
- When I mark a task as complete by clicking on the checkbox, the state of the checkbox does not persist. Please fix your project so that the state of the task checkbox persists.
### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by Forison — APPROVED** [review]
# Status::Approved✔️.
### Good job on addressing the issue raised in the previous review, kindly go ahead and merge the branch and proceed to the next milestone🤗.
> Nice UI by the way💪!!!
#### Happy coding
Addo Forison

**Pull request merged**